index.html

```diff
@@ -0,0 +1,641 @@
+        
+        <iframe
+          src="https://carbon.now.sh/embed/ozua4VaVE1DBw7M8bUdS"
+          style="width: 100%; height: 205px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+
+        <!-- Code snippet from Carbon for JS of component library -->
+        <h3>JS</h3>
+        <p>Some of our components require the use of JavaScript and icons to function and show some messages properly. Place the following <code>script</code> near the end of your page inside <code>body</code> tag.</p>
+        <br><br><br><br>
+
+        <iframe
+          src="https://carbon.now.sh/embed/LBGXwkmzEIht85xUeVpj"
+          style="width: 100%; height: 242px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+      </section>
+
+      <section id="alert">
+        <h3 class="header">Alerts</h3>
+        
+        <!-- Code snippet from Carbon for alerts -->
+        <div class="alert alert-primary">
+          <ion-icon class="alert-icon" name="information-circle"></ion-icon>
+          <strong>Primary alert.!!</strong>
+          <span class="alert-close"><ion-icon name="close"></ion-icon></span>
+        </div>
+
+        <div class="alert alert-secondary">
+          <ion-icon class="alert-icon" name="information-circle"></ion-icon>
+          <strong>Secondary alert.!!</strong
+          ><span class="alert-close"><ion-icon name="close"></ion-icon></span>
+        </div>
+
+        <div class="alert alert-success">
+          <ion-icon class="alert-icon" name="checkmark-circle"></ion-icon>
+          <strong>Success alert.!!</strong>
+          <span class="alert-close"><ion-icon name="close"></ion-icon></span>
+        </div>
+
+        <div class="alert alert-danger">
+          <ion-icon class="alert-icon" name="alert-circle"></ion-icon>
+          <strong>Danger alert.!!</strong>
+          <span class="alert-close"><ion-icon name="close"></ion-icon></span>
+        </div>
+        <br><br><br><br>
+
+        <iframe
+          src="https://carbon.now.sh/embed/sCeK97JAL5OF3ToqSDDF"
+          style="width:100%; height: 610px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+      </section>
+
+      <section id="badge">
+        <h3 class="header">Badges</h3>        
+       
+       <!-- Code snippet from Carbon for badges -->
+       <span class="badge bg-primary">Primary</span>
+       <span class="badge bg-secondary">Secondary</span>
+       <span class="badge bg-success">Success</span>
+       <span class="badge bg-danger">Danger</span>
+       <br><br><br><br>
+
+        <iframe
+          src="https://carbon.now.sh/embed/JgDpLHCQVecwDunkegPk"
+          style="width: 100%; height: 260px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
```

Comment:
Here in iframe, style is same for every iframe, so what you could do is , give a class name to each iframe and then refer it from css file.

Reply:
Hi roshan,
thanks for reviewing it. I had refactored the points which you have mentioned for last one i will keep in mind from next time coz it has been used multiple places so have to refactor whole code.